doc/examples/gestures/lib/main_multitap_advanced.dart

```diff
@@ -70,12 +72,14 @@ class MyGame extends Game with MultiTouchTapDetector, MultiTouchDragDetector {
   }
 
   void onPanEnd(DragEndDetails details) {
-    _panRect = Rect.fromLTRB(
-      _start.dx,
-      _start.dy,
-      _end.dx,
-      _end.dy,
-    );
+    if (_start != null && _end != null) {
+      _panRect = Rect.fromLTRB(
+        _start!.dx,
```

Comment:
do we need ! if we already asserted _start != null?

Reply:
well, `flutter analyze` fails to see that. it happens in some cases. I still didn't grasp when it does that completely but in this case, we need to be explicit.